src/content/smart-bulletins/036.md

```diff
@@ -0,0 +1,51 @@
+
+## Introduction
+
+This Smart Bulletin provides operational guidance regarding Public Law 116-160, the “Charging Helps Agencies Realize General Efficiencies” Act or “CHARGE” Act. This statute requires the U.S. General Services Administration (GSA) to issue guidance on the use of charge cards to pay for charging federally-owned or managed Electric motor Vehicles (EVs) used for official Government business at commercial charging stations. Additionally, this bulletin provides guidance on the use of GSA SmartPay payment solutions for payment of rental car EV charging related to travel on official business. Note that unless otherwise indicated, the mention of specific commercial charging network names in this bulletin is for information purposes only, is not intended as an endorsement of any kind, and does not indicate a contractual relationship with GSA.
+
+## Summary
+
+Participating agencies/organizations use GSA SmartPay charge cards to obtain approximately $30 billion in products and services annually in support of their missions through approximately 100 million purchase, travel and fleet transactions. The CHARGE Act emphasizes the existing ability for agencies/organizations to use GSA SmartPay charge cards to pay for charging services for EVs at commercial charging facilities. At the time of this guidance being issued, GSA SmartPay contractor bank charge cards, including WEX and Voyager fleet cards, can be used to pay for charging services at ChargePoint, Inc. commercial charging stations. Other charging station networks currently only accept Visa & MasterCard branded cards through their respective payment apps and terminals. Certain EVgo chargers do include card readers for walk-up payment, but not all. As the electric vehicle charging industry continues to expand, the Center for Charge Card Management (CCCM) and our GSA SmartPay contractor banks will pursue additional relationships with charging station providers to increase access and acceptance at additional charging networks for federal agencies. Use of GSA SmartPay charge cards for both per transaction and charging subscriptions is permissible consistent with agency policy, applicable charge card spending limit(s), the Federal Acquisition Regulation and other applicable rules/regulations. 
+
+## Action
+
+Each Federal agency may, for Government owned or managed EVs used in accordance with 31 U.S.C 1344, “Passenger Carrier Use,” (1) allow for the vehicles to be charged at a commercial EV charging station and (2) pay for that charging with a GSA SmartPay charge card. The type of GSA SmartPay charge card to be used (purchase, travel, integrated and/or fleet) to complete the charging transaction must comply with agency policy. GSA SmartPay charge cards may also be used to pay for commercial rental car EV changing when an employee is in official travel status. Additional operational considerations for use of charge cards pursuant to the CHARGE Act are provided below:
```

Comment:
Each Federal agency may, for Government-owned or managed EVs used in accordance with 31 U.S.C 1344, “Passenger Carrier Use,” (1) allow for the vehicles to be charged at a commercial EV charging station and (2) pay for that charging with a GSA SmartPay charge card. The type of GSA SmartPay charge card to be used (purchase, travel, integrated, and/or fleet) to complete the charging transaction must comply with agency policy. GSA SmartPay charge cards may also be used to pay for commercial rental car EV charging when an employee is in official travel status. Additional operational considerations for use of charge cards pursuant to the CHARGE Act are provided below: